src/Executor/ExecutionResult.php

```diff
@@ -142,10 +142,14 @@ public function toArray(int $debug = DebugFlag::NONE) : array
                 return array_map($formatter, $errors);
             };
 
-            $result['errors'] = $errorsHandler(
+            $formattedErrors = $errorsHandler(
```

Comment:
Not just formatted, also filtered etc.

```suggestion
            $handledErrors = $errorsHandler(
```